src/Humanizer.Tests/TimeSpanHumanizeTests.cs

```diff
@@ -82,8 +82,8 @@ public void TimeSpanWithMaxTimeUnit(int ms, string expected, TimeUnit maxUnit)
         }
 
         [Theory]
-        [InlineData(0, 3, "no time")]
```

Comment:
This was intended for cases when for example a post is just added and time span is 0 milliseconds so it would make sense to show 'just now'

But for what I understand from your comment sometimes users just want to translate such a time span to 'no time'

Would you like me to revert it back?